src/util/apiUtils.js

```diff
@@ -270,10 +275,13 @@ export function parseRequestHeaders(request) {
 export function parseRequestQueries(request) {
 	const {
 		method,
+		mime,
```

Comment:
easy: don't. 🙂 

the query system is intended to make you think about REST as little as possible - from the PR description:

> The current code checks whether `query.params` is a `FormData` object, and will treat the query as a multipart/form-data request.

I'll add more docs for this, but you should never have reason to manually managing content-type.